src/github/filterStaleBranches.js

```diff
@@ -1,12 +1,13 @@
 const { github } = require('../config');
 
-module.exports = ({ nodes, excludedBranches = github.exclude }) => {
+// filters stale, default and excluded branches
+module.exports = ({ nodes, defaultBranch = null, excludedBranches = github.exclude }) => {
   const past = new Date();
   past.setMonth(past.getMonth() - 3);
 
   const staleBranches = nodes
     .filter(({ target: { committedDate } }) => Date.parse(committedDate) < past)
-    .filter(({ name }) => !excludedBranches.includes(name));
+    .filter(({ name }) => ![...excludedBranches, defaultBranch].filter(Boolean).includes(name));
```

Comment:
What is `.filter(Boolean)` for? It seems to be to prevent some edge cases.

Reply:
yea, if defaultBranch is ever null; it's probably never null in real life but in the params of this function it's defaulting to null so the function could be called with a null.

Reply:
I see. Makes sense.